packages/checkbox/src/checkbox.vue

```diff
@@ -25,6 +25,7 @@
       <input
         v-if="trueLabel || falseLabel"
         v-model="model"
+        :checked="isChecked"
```

Comment:
Note the `checked` prop here. Vue rc 9 no longer treats non-boolean `model` as `true` in checkbox's v-model